### PR TITLE
Mass Notification Feature

### PR DESCRIPTION
**Pull request by dorozco94**

### CODE REVIEW

File: source/includes/_notifications.md [hunk 1]
```diff
@@ -234,6 +237,7 @@ name       | string  | Name for notification
 entity_type| string  | Entity type for notification
 action     | string | Notification action [account_created, recover_password, attendee_created, attendee_rejected, attendee_confirmed, welcome_email]
 conditions  | json | Conditions for notifications
+mass_notification  | booleand | Flag to identify mass notification
```
Comment (focampo92) [inline source/includes/_notifications.md:240]: boleand o **boolean**?, 

la descripción puede ser: 
Flag to identify if the notification that is going to be created is massive

File: source/includes/_notifications.md [hunk 2]
```diff
@@ -339,6 +346,7 @@ name       | string  | Name for notification
 entity_type| string  | Entity type for notification
 action      | string | Notification action [account_created, recover_password, attendee_created, attendee_rejected, attendee_confirmed, welcome_email]
 conditions  | json | Conditions for notifications
+mass_notification  | booleand | Flag to identify mass notification
```
Comment (focampo92) [inline source/includes/_notifications.md:349]: Aplica comentario anterior.

File: source/includes/_notifications.md [hunk 3]
```diff
@@ -405,6 +414,130 @@ event_uri | string  | The event_uri for the desired event
...
+
+```http
+HTTP/1.1 404 Not Found
+{
+  {
+    "message": "Couldn't find Notification"
+  }
+}
+```
+This endpoint get the mass notifications stats and return it
```
Comment (focampo92) [inline source/includes/_notifications.md:527]: This endpoint allows you to obtain the progress and stats of a massive notification after the user sent it.

**Review by focampo92 — CHANGES_REQUESTED** [review]

File: source/includes/_notifications.md (hunk 1 above)
Comment (dorozco94) [reply source/includes/_notifications.md]: Done

File: source/includes/_notifications.md (hunk 2 above)
Comment (dorozco94) [reply source/includes/_notifications.md]: Done

File: source/includes/_notifications.md (hunk 3 above)
Comment (dorozco94) [reply source/includes/_notifications.md]: Done

File: source/includes/_notifications.md [hunk 4]
```diff
@@ -724,4 +857,4 @@ Parameter | Type | Description
 --------- | ---- | -----------
 event_uri | string | The event_uri for the desired event.
 notification_id | integer | The id for the desired notification
-id | integer | The id for the notification language
\ No newline at end of file
+id | integer | The id for the notification language
```
Comment (PierreQuintero-eventtia) [inline source/includes/_notifications.md:860]: Id for the desired notification language

File: source/includes/_notifications.md [hunk 5]
```diff
@@ -405,6 +414,130 @@ event_uri | string  | The event_uri for the desired event
...
+***HTTP Request***
+
+`GET /v1/events/:event_uri/notifications/:id/stats`
+
+***Path Parameters***
+
+Parameter |  Type   | Description
+--------- | ------- | -----------
+event_uri | string  | The event_uri for the desired event
+   id     | integer | The id for the desired notification
```
Comment (PierreQuintero-eventtia) [inline source/includes/_notifications.md:538]: Id for the desired notification

File: source/includes/_notifications.md [hunk 6]
```diff
@@ -405,6 +414,130 @@ event_uri | string  | The event_uri for the desired event
...
+
+***HTTP Request***
+
+`GET /v1/events/:event_uri/notifications/:id/stats`
+
+***Path Parameters***
+
+Parameter |  Type   | Description
+--------- | ------- | -----------
+event_uri | string  | The event_uri for the desired event
```
Comment (PierreQuintero-eventtia) [inline source/includes/_notifications.md:537]: event_uri for the desired event

**Review by PierreQuintero-eventtia — CHANGES_REQUESTED** [review]

**Review by focampo92 — APPROVED** [review]

File: source/includes/_notifications.md (hunk 4 above)
Comment (dorozco94) [reply source/includes/_notifications.md]: Done

File: source/includes/_notifications.md (hunk 5 above)
Comment (dorozco94) [reply source/includes/_notifications.md:538]: Done

File: source/includes/_notifications.md (hunk 6 above)
Comment (dorozco94) [reply source/includes/_notifications.md]: Done

**Review by PierreQuintero-eventtia — APPROVED** [review]

**Pull request merged**